src/main/java/org/folio/entitlement/integration/kafka/SystemUserModuleEventPublisher.java

```diff
@@ -35,17 +40,11 @@ protected String getResourceName() {
   /**
    * Creates a system user event for provided {@link ModuleDescriptor} object.
    *
-   * @param moduleDescriptor - {@link ModuleDescriptor} object
+   * @param systemUser - {@link ModuleDescriptor} object
```

Comment:
Why remove parameter moduleDescriptor from javadoc? It's still there in the method.